core/src/main/java/org/springframework/security/access/prepost/PrePostAdviceReactiveMethodInterceptor.java

```diff
@@ -53,6 +53,7 @@
  * @author Eleftheria Stein
  * @since 5.0
  */
+@Deprecated
```

Comment:
I think we shouldn't deprecate it for now because we decided to add the `authorizationManager` flag until https://github.com/spring-projects/spring-framework/issues/22462 is fixed.